src/devices/legrand.ts

```diff
@@ -697,6 +697,52 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['NLIS - Light switch'],
+        model: 'NLIS - Light switch',
```

Comment:
Please use the model numbers written on the device/product page (also use it in https://github.com/Koenkk/zigbee2mqtt.io/pull/3030)